.github/workflows/deploy-snapshots.yml

```diff
@@ -36,4 +36,4 @@ jobs:
         run: mvn -B deploy --file pom.xml
         env:
           BINTRAY_USERNAME: ugandaemr-ci
-          BINTRAY_PASSWORD: aGJUeZZbGSy3aGbZc49aEf
+          BINTRAY_PASSWORD: ${{ secrets.BINTRAY_ACCESS_PASSWORD }}
```

Comment:
@solemabrothers Why use secrets.BINTRAY_ACCESS_PASSWORD here and BINTRAY_PASSWORD elsewhere

Reply:
@ssmusoke BINTRAY_PASSWORD is the environment variable while secrets.BINTRAY_ACCESS_PASSWORD is the value of the secret created.

Reply:
@solemabrothers then name the secret secrets.BINTRAY_PASSWORD to match the environment variable

Reply:
Done

Reply:
@solemabrothers This is not done - check